lib/bamboo/plug/sent_email_api/sent_email_api_plug.ex

```diff
@@ -0,0 +1,112 @@
+defmodule Bamboo.SentEmailApiPlug do
+  use Plug.Router
+  alias Bamboo.SentEmail
+
+  @moduledoc """
```

Comment:
👏 Yes, this documentation is great and super clear! I think it helps justify this being separate and clearly states out the case where you'd use these endpoints, vs. the ones in the `Bamboo.SentEmailViewerPlug`. Thanks for the level of detail and examples in here!